### ISSUE TITLE
Prisma is not finding the JobWhereInput for some reason

### ISSUE THREAD

**Issue by weisisheng** [codinginflow/nextjs-job-board#1]
About 2:07  in to video, when I try to create the search filter and 'where' segments, I am getting the squiggly line TS errors that "Prisma.JobWhereInput" has no exported members.  Checked the validation file is correct.

Here is exact error:

Property 'JobWhereInput' does not exist on type 'typeof import("/home/foouser/nextjs-world/coding-in-flow-tutorials/nextjs-job-board/node_modules/@prisma/client/index"

I am more familiar with drizzle-orm so this is a new command and error for me.  

Thanks in advance for any assistance.  Already checked copilot and google.

next.js 14.10.0
@prisma/client 5.7.1
prisma 5.7.1
react ^18

**Comment by codinginflow** [comment]
Did you generate the prisma client with `npx prisma generate`?

**Comment by weisisheng** [comment]
I did, let me try it again (think I can do this without harm)

**Comment by weisisheng** [comment]
Yes, re-ran prisma generate, no errors, restarted typescript server.  puzzled.  will give it a look-see over weekend/

**Comment by fhub-1** [comment]
> Yes, re-ran prisma generate, no errors, restarted typescript server. puzzled. will give it a look-see over weekend/

did you mange the error
